Neuro/Vision/VisionHandler.cs

```diff
@@ -14,12 +16,14 @@ public class VisionHandler : MonoBehaviour
     public VisionHandler(IntPtr ptr) : base(ptr) { }
 
     public Vector2 DirectionToNearestBody { get; set; }
-    public readonly Dictionary<byte, PlayerRecord> PlayerRecords = new(); // TODO: Store this data using a monobehaviour on the player
+    
+    [HideFromIl2Cpp]
+    public Dictionary<PlayerControl, PlayerRecord> PlayerRecords { get; } = new(Il2CppEqualityComparer<PlayerControl>.Instance);
```

Comment:
> // TODO: Store this data using a monobehaviour on the player

No?

Reply:
Is there really any point in doing that here? GetComponent is going to be much slower than a dictionary access

Reply:
Well we have both PlayerRecord and LastSeenPlayer